scripts/common/text_files/country_demonyms.txt

```diff
@@ -0,0 +1,32 @@
+Djiboutians
+Eritreans
+Ethiopians
+Somalis
+Sudanese
+Malagasy
+Mauritians
+Comorans
+Seychellois
+Ugandans
+Rwandans
+Burundians
+Kenyans
+Tanzanians
+Mozambicans
+Malawians
+Zambians
+Zimbabweans
+Egyptians
+Bahrainis
+Iranians
+Iraqis
+Jordanians
+Kuwaitis
+Lebanese
+Omanis
+Palestinians
+Qataris
+Syrians
+Turks
+Emiratis
+Yemenis
```

Comment:
Consider compressing the data files with gzip so that git doesn't try to line-diff them. Using gzip from Python is just about as easy as using normal files (`import gzip; with gzip.open(...) as fobj: ...`)

Reply:
done!